doc/guide/installation.rst

```diff
@@ -38,6 +44,11 @@ The easiest way to keep your suite updated is to use
           # ...
           extensions:
             SensioLabs\Behat\PageObjectExtension: ~
+            Behat\MinkExtension:
```

Comment:
As far as behat.yml is concerned, I've always seen this indented with 2 spaces so far.